addons/l10n_uy/models/account_tax.py

```diff
@@ -0,0 +1,11 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+
+    _inherit = "account.tax"
+
+    l10n_uy_tax_category = fields.Selection([
+        ('vat', 'VAT'),
```

Comment:
@jco-odoo for now we just added VAT as per the other taxes and witholds we need to go deeper in the analysis to determined what other options will be required in the selection once that we add those records in the data. 

cc. @zaoral @fvz-odoo

Reply:
@MarianaLuzTorres @zaoral one last minor comment here, I suggest to add a tooltip, so it's clearler for the user what's the poporpuse of this field, something like this "UY: Use to group the transactions in the Financial Reports required by DGI" 

What do you think?   
cc. @fvz-odoo